README.md

```diff
@@ -16,18 +16,31 @@ version of Firefox.
 Installation
 ------------
 
-OpenWPM has been developed and tested on Ubuntu 14.04/16.04. An installation
```

Comment:
The first sentence here should remain, since we do not add tests for Mac and Windows.